io.github.MakovWait.Godots.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.MakovWait.Godots.desktop</id>
```

Comment:
```suggestion
  <id>io.github.MakovWait.Godots</id>
```

Please tell me which doc told you to add .desktop so I get that one fixed too....

Reply:
Ah, I believe that was just a typo.